RedHatInsights/ansible-collections-insights
# ci: simplify ansible-test test

Simplify the `ansible-test` test a bit:
- use `working-directory` more for the steps
- use a simpler checkout directory layout
- minor tweak to comments
- drop dead condition

See the individual commit messages for longer explanations; there should be no behaviour change in the `ansible-test` workflow.
